python/usepyhf/confidence.py

```diff
@@ -0,0 +1,39 @@
+import pyhf
+import numpy as np
+from usepyhf import util
+
+def upper_limit(workspace:pyhf.Workspace):
+    """
+    Calculates the observed upper limit of the POI at the 95% CL.
+    This function also prints the expected limit median and +/- [1, 2] sigma.
+    """
+    #Extract model and data
+    model = workspace.model()
+    data = workspace.data(model)
+    
+    #Find limits
+    poi_index = model.config.poi_index
+    poi_min, poi_max = model.config.suggested_bounds()[poi_index]
```

Comment:
Can the min and max be taken from the scan range variable in the config manager if it's not None? https://github.com/histfitter/histfitter/blob/master/python/configManager.py#L127